pom.xml

```diff
@@ -601,6 +612,28 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>enforce-jdbi</id>
+                        <goals>
+                            <goal>enforce</goal>
+                        </goals>
+                        <configuration>
+                            <rules>
+                                <!-- Ensure consistency -->
+                                <dependencyConvergence />
```

Comment:
We already have a different plugin for this purpose from `basepom`. The enforcer plugin doesn't provide very helpful messages, unfortunately.

Reply:
I'm unfamiliar with `basepom`, however it clearly doesn't handle dependency convergence the same way as the enforcer plugin as it allowed a circular dependency reference to make it through (breaking downstream things, like DropWizard, that rely on dependency convergence to prevent bugs/issues). Perhaps there is a way to configure it so that it plays ball with the standard maven plugins used by projects that depend on JDBI3?